protractor.conf.js

```diff
@@ -1,30 +0,0 @@
-// Protractor configuration file, see link for more information
```

Comment:
Why did Protractor get removed? Are we using a different solution for e2e tests? Please clarify